src/page-multi-facility/HistoricalCasesChart/BarChartTooltip.tsx

```diff
@@ -47,19 +50,22 @@ const TooltipSubtitle = styled.div`
 
 const BarChartTooltip: React.FC<TooltipProps> = ({ summary }) => {
   const summaryData = summary[0];
-  const { data, value } = summaryData;
+  const { data } = summaryData;
   return (
     <TooltipContainer>
       {data.missing ? (
         <TooltipTitle>No data</TooltipTitle>
       ) : (
-        <TooltipTitle>{value} cases</TooltipTitle>
+        <>
+          <TooltipTitle>{data.cases} cases</TooltipTitle>
+          <TooltipTitle>{data.population + data.cases} residents</TooltipTitle>
```

Comment:
It's not obvious to me why this part of the tooltip would sum population and cases. Shouldn't this just be `data.population`?

Reply:
Maybe I don't understand exactly what these variables refer to, because it seems to just display the population count in my local testing...

Reply:
The design requirement is for an overlay bar chart (i.e. both bars starting from the x axis). Semiotic supports a stacked bar chart where the second bar will start from the top of the first. So in order to get it to work like an overlay, I changed the second bar data to be the population - confirmed cases.
Since the tooltip needs to continuing showing the total population, I need to add these here. I can rename this to be non-cases or something like that to be clearer.

Reply:
Thanks for the explanation, @SoumyaMahadevan! I understand now.